app/views/circuitverse_assets/_search_bar.html.erb

```diff
@@ -7,3 +7,12 @@
   <% end %>
   </div>
 </div>
+<script>
+  $(document).ready(function() {
+    const searchType = $("#search_type");
+    const searchInput = $("#q");
+    searchType.on("change", function() {
+      searchInput.focus();
+    });
+  });
```

Comment:
@shauryag2002 We are not using jQuery anymore, as you are a new contributor would recommend you to implement a vanilla javascript-typed code or use `stimulus` for this, would be a good learning for you

https://stimulus.hotwired.dev/